chart/instana-agent/values.yaml

```diff
@@ -81,3 +170,73 @@ agent:
 cluster:
   # cluster.name represents the name that will be assigned to this cluster in Instana
   name: null
+
+leaderElector:
+  image:
+    # leaderElector.image.name is the name of the container image of the leader elector.
+    name: instana/leader-elector
+    # leaderElector.image.digest is the digest (a.k.a. Image ID) of the leader elector container image; if specified, it has priority over leaderElector.image.digest, which will be ignored.
+    #digest:
+    # leaderElector.image.tag is the tag name of the agent container image; if leaderElector.image.digest is specified, this property is ignored.
+    tag: 0.5.10
+  port: 42655
+
+# openshift specifies whether the cluster role should include openshift permissions and other tweaks to the YAML.
+# The chart will try to auto-detect if the cluster is OpenShift, so you will likely not even need to set this explicitly.
+# openshift: true
+
+rbac:
+  # Specifies whether RBAC resources should be created
+  create: false
+
+service:
+  # Specifies whether to create the instana-agent service to expose within the cluster the Prometheus remote-write, OpenTelemetry GRCP endpoint and other APIs
+  # Note: Requires Kubernetes 1.17+, as it uses topologyKeys
+  create: false
+
+opentelemetry:
+  enabled: false # If true, it will also apply `service.create=true`
+
+prometheus:
+  remoteWrite:
+    enabled: false # If true, it will also apply `service.create=true`
+
+serviceAccount:
+  # Specifies whether a ServiceAccount should be created
+  create: false
```

Comment:
This one (and the `rbac`) should not be used by the customer, right? If they enable then the deployment fails?
Perhaps then we should not expose them at all. Or put them in some separate place and make it _really_ clear customers should not change those params.

Reply:
Or is this handled correctly by this `schema.yaml` file and customers wouldn't touch this `values.yaml` anyhow?
(then still at least some comment might be helpful for the next time we need to update with our latest Helm chart)

Reply:
Should be handled by Schema because you can only change attributes which are available in the Schema.